apps/speed.c

```diff
@@ -1463,9 +1521,21 @@ int speed_main(int argc, char **argv)
         {"X25519", NID_X25519, 253},
         {"X448", NID_X448, 448}
     };
+    static const struct {
+        const char *name;
+        unsigned int nid;
+        unsigned int bits;
+        unsigned int siglen;
+    } test_ed_curves[] = {
+        /* EdDSA */
+        {"Ed25519", NID_ED25519, 253, 64},
+        {"Ed448", NID_ED448, 448, 114}
```

Comment:
I'm not sure what the "bits" arg represents. Key size? Security bits? Anyway the values given here don't seem to completely agree with the values here (at least the Ed448 bits value seems different):

https://github.com/openssl/openssl/blob/2167640b0bf76ec50a397dd90444b97c242e3f04/crypto/ec/ecx_meth.c#L19-L28

Reply:
Represents the key size. it seems Ed448 bits should be changed to `ED448_BITS` in speed